tests/ILVerify/ilverify_FSharp.Compiler.Service_Debug_netstandard2.0.bsl

```diff
@@ -96,20 +96,3 @@
 [IL]: Error [StackUnexpected]: : FSharp.Compiler.IO.FileSystemUtils::trimQuotes(string)][offset 0x00000014][found Char] Unexpected type on the stack.
 [IL]: Error [StackUnexpected]: : Internal.Utilities.Library.String::lowerCaseFirstChar(string)][offset 0x0000003F][found Char] Unexpected type on the stack.
 [IL]: Error [StackUnexpected]: : Internal.Utilities.Library.Array+loop@276-4::Invoke(int32)][offset 0x00000012][found Byte] Unexpected type on the stack.
-[IL]: Error [StackUnexpected]: : Microsoft.FSharp.Collections.ArrayModule+Parallel::Choose([FSharp.Core]Microsoft.FSharp.Core.FSharpFunc`2<!!0,Microsoft.FSharp.Core.FSharpOption`1<!!1>>, !!0[])][offset 0x000000A0][found Byte] Unexpected type on the stack.
```

Comment:
This part I do not get - why are they removed?
(not that I complain, it's good to have less of then - is it because of the entry added to dotnet-tools.json forcing a version?)

Reply:
The version of ilverify is the remaining version 9.0.  I assume they fixed a bug between preview and release.